_posts/languages/ruby/2018-02-23-getting-started-with-ruby.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Getting Started with Ruby
+modified_at: 2018-02-23 00:00:00
+category: getting-started
+tags: ruby sinatra tutorial getting-started-tutorial
+index: 3
+permalink: /languages/ruby/getting-started-with-ruby/
+---
+
+## Installing the tools
+
+This tutorial will show you how to deploy a ruby app in minutes.
+
+Note: This tutorial does not cover the rails installation, if you want to start a rails project use our [getting started with rails tutorial]({% post_url languages/ruby/2015-02-17-getting-started-with-rails %})
+
+To be able to follow this tutorial, we assume that you have:
+
+* A [Scalingo account](https://scalingo.com/users/signup)
+* Ruby version 2.3.5+ -- see the gorails setup guide [window](https://gorails.com/setup/windows/10), [mac](https://gorails.com/setup/osx/10.13-high-sierra), [linux](ihttps://gorails.com/setup/ubuntu/17.10)
+* [Bundler](http://bundler.io/) installed locally: `gem install bundler`
+* The [Scalingo cli](http://cli.scalingo.com/)
+
+## Bootstrap your project
+
+Let's start by creating a new ruby application:
+
+```bash
+$ mkdir my-ruby-app
+$ cd my-ruby-app
+```
+
+And create your `Gemfile`
+
+```bash
+$ bundle init
+```
+
+This will create a file name `Gemfile`. This file will be used for describing gem dependencies for your ruby application.
+
+
+## Create your ruby application
+
+<blockquote class="bg-info">
```

Comment:
no fucking blockquote

Reply:
What should i use instead.
Specify it in the fucking README

Reply:
Note should be in `<aside class="note" markdown="1"></aside>`

Reply:
So far it's present here:  https://github.com/Scalingo/homepage (README), I'd like @yannski to create some doc on `scalingo-styles` if possible to let us know this kind of things

Reply:
Right ...

![2018-02-23_143x54_scrot](https://user-images.githubusercontent.com/3317829/36594657-590e830a-189f-11e8-8fd3-542a0be345a0.png)

![2018-02-23_395x69_scrot](https://user-images.githubusercontent.com/3317829/36594663-5fd4bb28-189f-11e8-985d-be6a904f387c.png)

Reply:
You syntax highlighter is not great =P

Reply:
Think of the future @johnsudaar , the new template is coming